PowerAssert.MultipleAssertions.PolyAssertException: PolyAssert encountered 1 failures:
> Sometimes you do want to end the test early after all
> So just call StopIfErrorsHaveOccurred (behaves the same as disposing the PolyAssert)
============================================================
ERROR in EndToEndTest.PolyAssertCanFinishEarly at :<line>:
============================================================
Wah wah
============================================================


   at PowerAssert.MultipleAssertions.PolyAssert.StopIfErrorsHaveOccurred()
   at PowerAssert.MultipleAssertions.PolyAssert.Dispose()
   at PowerAssertTests.Approvals.EndToEndTest.PolyAssertCanFinishEarly()